feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/provider/ChangeableFeeLoaderProvider.kt

```diff
@@ -77,11 +77,10 @@ internal open class ChangeableFeeLoaderProvider<F : GenericFee>(
 
     private val supportCustomFeeFlow = MutableStateFlow(configuration.initialState.supportCustomFee)
 
-    private val availableFeeAssetsFlow: Flow<Map<Int, Chain.Asset>> = combine(supportCustomFeeFlow, selectedTokenFlow) { supportCustomFee, selectedToken ->
-        if (!supportCustomFee) return@combine emptyMap()
+    private val availableFeeAssetsFlow: Flow<Boolean> = combine(supportCustomFeeFlow, selectedTokenFlow) { supportCustomFee, selectedToken ->
```

Comment:
availableFeeAssetsFlow should be renamed if its now a Boolean

Reply:
Agree. Fixed